### PR TITLE
Content fixes

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
https://trello.com/c/m6JLh8pq/16-incorrect-content

### CODE REVIEW

File: _unused/lab-swedlow.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <div class="callout large primary">
             <div class="row column text-center">
                 <h1>Swedlow Lab</h1>
-                <p>Prof Jason Swedlow, Wellcome Trust Centre for Gene Regulation &amp; Expression, University of Dundee, UK</p>
+                <p>Prof Jason Swedlow, Discovery Centre for Translational &amp; Interdisciplinary Research, University of Dundee, UK</p>
```
Comment (sbesson) [inline _unused/lab-swedlow.html]: Although the Discover Centre is the correct name for our building, we should rather use 

```
Division of Computational Biology, School of Life Sciences, University of Dundee
```

for the affiliation across the board.

**Comment by sbesson** [comment]
Changes are sensible and look good to merge. Only unknown is whether there is a proper class for highlighting the syntax under `explore/index.html`. @lunson: do you have more hindsights?

**Comment by rleigh-codelibre** [comment]
@lunson Wrap or indent?  It's verbatim preformatted text, with explicit indentation which is why I also added `<pre>` tags around the `<code>`.

Could the highlighting be due to missing CSS?

**Comment by lunson** [comment]
@sbesson The code class should be "html" not "sh".

**Comment by rleigh-codelibre** [comment]
@lunson Why is the code class `html` when the content here is not HTML?  It's shell script; is there a class for this we could use instead?  Is there a list of the possible classes for the `<code>` element?

**Comment by lunson** [comment]
@rleigh-codelibre It was just a style name that stuck. You can call the class whatever you want, but be sure to change the CSS selector in openmicroscopy.css (line 57) to match. For example, if you change it to `<code class="shell-script">`, the selector in the CSS file should be changed from `code.html` to `code.shell-script`.

**Comment by rleigh-codelibre** [comment]
@lunson Ah, OK.  I was assuming this hooked into the jekyll Rouge or Markdown code formatting/highlighting stuff.

**Comment by rleigh-codelibre** [comment]
Should all be fixed up now.

**Comment by lunson** [comment]
Thanks @rleigh-codelibre!

**Pull request merged**